src/uwtools/tests/drivers/test_esg_grid.py

```diff
@@ -0,0 +1,134 @@
+# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
+"""
+EsgGrid driver tests.
+"""
+from unittest.mock import DEFAULT as D
+from unittest.mock import patch
+
+import f90nml  # type: ignore
+import yaml
+from pytest import fixture
+
+from uwtools.drivers import esg_grid
+from uwtools.scheduler import Slurm
+
+# Driver fixtures
+
+
+@fixture
+def config(tmp_path):
+    return {
+        "esg_grid": {
+            "execution": {
+                "batchargs": {
+                    "export": "NONE",
+                    "nodes": 1,
+                    "stdout": "/path/to/file",
+                    "walltime": "00:02:00",
+                },
```

Comment:
Based on Christina's open PR, some of them might not be needed in the `execution` block. Specifically `export` and `stdout`. I also have `nodes` here instead of `cores`. Not sure which values are appropriate.

Reply:
Per [our docs](https://uwtools.readthedocs.io/en/main/sections/user_guide/yaml/execution.html#batchargs), the only required entry is `walltime`, but I just tested with the `docs/shared/ungrib.yaml` and schema validation passes without it. There might be a runtime error later, but I'm not immediately seeing where it would come from. Probably not for this PR, but we should consider making `walltime` schema-required if we mean what we say in the docs.

Reply:
Oh, maybe `_validate_props()` in `uwtools.scheduler` enforces this. It looks like that logic also requires `account`, but that is provided in the `platform:` block, which has its own schema that does require `account` if `scheduler` is specified. So, maybe our `execution` `batchargs` subschema should require `walltime`... But maybe we should make a note of this rather than hold up this PR for it.

Reply:
Sounds great, would the best way to document this to be to open an issue and tag this discussion?